Lib/glyphsLib/classes.py

```diff
@@ -1261,6 +1250,9 @@ def __init__(self):
         self._custom2 = None
         self.italicAngle = 0.0
         self.customValue = 0.0
+        self._userData = None
+        self.xHeight = None
+        self.capHeight = None
```

Comment:
@schriftgestalt What are the defaults for `xHeight` and `capHeight`? I guess it's not `0` because some files specify a value of `0`

Reply:
@belluzj what are the defaults when you create an empty new font? (I don't have my mac at hand)